sourcepolicy/sourcepolicy.go

```diff
@@ -0,0 +1,28 @@
+// Package sourcepolicy provides the source policy types
+package sourcepolicy
+
+import binfotypes "github.com/moby/buildkit/util/buildinfo/types"
+
+// SourceType defines the source type.
+type SourceType = binfotypes.SourceType
+
+// Source is a subset of github.com/moby/buildkit/util/buildinfo/types.Source .
```

Comment:
If we extend this in the future to allow specifying policy like "is this signed", or even to "here's the signature" of the ref, then I think these structs would probably diverge?

Reply:
Changed "a subset of" to "similar to"